code/jack/html/lab4/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Source+Sans+Pro:wght@300&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
+    
+    <title>Burrito Order</title>
+</head>
+<body>
+    <header class="banner">
+        <div>
+            <h1>JACK'S BIG BURRITO PLACE</h1>
+        </div>
+    </header>
+    
+    <div class="main">
+        <div class="login">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/6/60/Burrito.JPG">
+            <h3>Login</h3>
+            <form action="https://webhook.site/5d056d77-ee17-4efc-b622-33fa3c7487c9" method="post">
+                <label for="name">Enter your name:</label>
+                <input type="text" name="name" id="name" value="name"><br>
```

Comment:
* For the 'input' tag, it is better to use 'placeholder' for placeholder text so the user doesn't have to delete the text prvided by the 'value' attribute:
    * Use 'placeholder' attribute instead of 'value' for name input element.